### PR TITLE
Update ckb to the latest develop

### PR DESCRIPTION
**Pull request by mohanson**

### CODE REVIEW

File: bins/src/main.rs [hunk 1]
```diff
@@ -173,14 +185,44 @@ fn main() {
         .parse()
         .expect("parse max cycle");
 
+    let script_version_u32: u32 = matches
+        .value_of("script-version")
+        .unwrap()
+        .parse()
+        .expect("parse vm version");
+    let script_version = if script_version_u32 == 0u32 {
```
Comment (xxuejie) [inline bins/src/main.rs]: I would suggest we let the user pick either `0` or `1`, and exit with an error if the user specifies `2` or other values.

File: Cargo.toml [hunk 2]
```diff
@@ -11,10 +11,12 @@ exclude = ["bins", "js"]
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-ckb-error = "0.42.0"
-ckb-traits = "0.42.0"
-ckb-types = "0.42.0"
-ckb-jsonrpc-types = "0.42.0"
+ckb-chain-spec = { git="https://github.com/nervosnetwork/ckb", branch = "develop" }
```
Comment (xxuejie) [inline Cargo.toml]: https://github.com/nervosnetwork/ckb/releases/tag/v0.100.0-rc1 Let's change here to this tag version(ideally, we can wait for those crates to be published to crates.io later today)

Comment (mohanson) [reply Cargo.toml]: This number is really big :)

Comment (xxuejie) [reply Cargo.toml]: Not as big as JRuby 9000: http://jruby.org/2015/07/22/jruby-9-0-0-0.html

Comment (doitian) [reply Cargo.toml]: I'll publish rc2 crates today.

**Review by xxuejie — APPROVED** [review]

**Pull request merged**